Src/Screens/DashboardScreen/Viewables/ExpandableRow.swift

```diff
@@ -56,10 +56,19 @@ extension ExpandableRow: Viewable {
         clippingView.clipsToBounds = true
 
         let expandableStackView = UIStackView()
+        expandableStackView.backgroundColor = .white
         expandableStackView.axis = .vertical
 
-        bag += expandableStackView.addArranged(content)
-
+        let contentWrapper = UIControl()
+        
+        bag += contentWrapper.add(content) { buttonView in
```

Comment:
`contentWrapperView` maybe instead of `buttonView`

Reply:
yeah that's more consistent 👍